hooks/basic/publish_files.py

```diff
@@ -382,7 +382,12 @@ def validate(self, task_settings, item):
         conflict_info = None
         if item.properties.is_sequence:
             seq_pattern = publisher.util.get_path_for_frame(item.properties.publish_path, "*")
-            seq_files = [f for f in glob.iglob(seq_pattern) if os.path.isfile(f)]
+            if seq_pattern:
+                seq_files = [f for f in glob.iglob(seq_pattern) if os.path.isfile(f)]
+            else:
+                # this means the publish_path doesn't follow SEQ pattern
+                # this could also mean it's an in-place publish and the path is not conformed to our pipeline
+                seq_files = []
```

Comment:
Instead of doing this, why don't you just wrap this entire validation block in the `if task_settings.get("publish_path_template")` conditional? Seems like a cleaner solution.